cli/lib/files.js

```diff
@@ -20,3 +25,7 @@ export const createFile = (filePath, content) => {
   createBaseFolderPathForFile(filePath);
   fs.writeFileSync(filePath, content);
 };
+
+export const rmDir = (path) => {
+  fs.rmSync(path, { recursive: true, force: true });
```

Comment:
force? 😅

Reply:
For once i acted according to the docs:
`node:47507) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will throw if path does not exist or is a file. Use fs.rm(path, { recursive: true, force: true }) instead`